scripts/templates/ldrddi.hpp.mako

```diff
@@ -19,6 +19,8 @@ from templates import helper as th
 #ifndef UR_LOADER_LDRDDI_H
 #define UR_LOADER_LDRDDI_H 1
 
+#include "ur_object.hpp"
```

Comment:
`#include "${x}_object.hpp"`. Change the copyright date here and the ldrddi.cpp.mako file.